rosa_tutorials/rosa-mobb-enabling-aws-efs-csi-driver-operator-rosa.adoc

```diff
@@ -0,0 +1,573 @@
+          --subnet-id $SUBNET --security-groups $SG \
+          --region $AWS_REGION \
+          | jq -r '.MountTargetId'); \
+       echo $MOUNT_TARGET; \
+done
+----
+
+. Creating a single-zone EFS
++
+[NOTE]
+====
+If you created a region-wide EFS mount, skip this step.
+====
++
+.. Select the first subnet to make your EFS mount in. This will, by default, select the same subnet your first node is in:
++
+[source,terminal]
+----
+   SUBNET=$(aws ec2 describe-subnets \
+     --filters Name=vpc-id,Values=$VPC Name=tag:Name,Values='*-private*' \
+     --query 'Subnets[*].{SubnetId:SubnetId}' \
+     --region $AWS_REGION \
+     | jq -r '.[0].SubnetId')
+   AWS_ZONE=$(aws ec2 describe-subnets --filters Name=subnet-id,Values=$SUBNET \
+     --region $AWS_REGION | jq -r '.Subnets[0].AvailabilityZone')
+----
+
+.. Create your zonal EFS file system:
++
+[source,terminal]
+----
+EFS=$(aws efs create-file-system --creation-token efs-token-1 \
+    --availability-zone-name $AWS_ZONE \
+    --region $AWS_REGION \
+    --encrypted | jq -r '.FileSystemId')
+echo $EFS
+----
+
+.. Create your EFS mount point:
++
+[source,terminal]
+----
+MOUNT_TARGET=$(aws efs create-mount-target --file-system-id $EFS \
+    --subnet-id $SUBNET --security-groups $SG \
+    --region $AWS_REGION \
+     | jq -r '.MountTargetId')
+echo $MOUNT_TARGET
+----
+
+.. Create a storage class for the EFS volume:
++
+[source,terminal]
+----
+cat <<EOF | oc apply -f -
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+    name: efs-sc
+provisioner: efs.csi.aws.com
+parameters:
+    provisioningMode: efs-ap
+    fileSystemId: $EFS
+    directoryPerms: "700"
+    gidRangeStart: "1000"
+    gidRangeEnd: "2000"
+    basePath: "/dynamic_provisioning"
+EOF
+----
+
+. Test
+
+.. Create a namespace:
++
+[source,terminal]
+----
+oc new-project efs-demo
+----
+
+.. Create a PVC
++
+[source,terminal]
+----
+cat <<EOF | oc apply -f -
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+    name: pvc-efs-volume
+spec:
+    storageClassName: efs-sc
+    accessModes:
+       - ReadWriteMany
+    resources:
+       requests:
+         storage: 5Gi
+EOF
+----
+
+.. Create a pod to write to the EFS volume:
++
+[source,terminal]
+----
+   cat <<EOF | oc apply -f -
+   apiVersion: v1
+   kind: Pod
+   metadata:
+    name: test-efs
+   spec:
+    volumes:
+      - name: efs-storage-vol
+        persistentVolumeClaim:
+          claimName: pvc-efs-volume
+    containers:
+      - name: test-efs
+        image: centos:latest
+        command: [ "/bin/bash", "-c", "--" ]
+        args: [ "while true; do echo 'hello efs' | tee -a /mnt/efs-data/verify-efs && sleep 5; done;" ]
+        volumeMounts:
+          - mountPath: "/mnt/efs-data"
+            name: efs-storage-vol
+   EOF
+----
++
+It might take a few minutes for the pod to be ready. If you see errors such as `Output: Failed to resolve "fs-XXXX.efs.us-east-2.amazonaws.com"` it likely means its still setting up the EFS volume.
```

Comment:
```suggestion
It might take a few minutes for the pod to be ready. If you see errors such as `Output: Failed to resolve "fs-XXXX.efs.us-east-2.amazonaws.com"`, it likely means it is still setting up the EFS volume.
```